api/checluster_conversion_from_test.go

```diff
@@ -361,8 +361,14 @@ func TestConvertFrom(t *testing.T) {
 					Value:    "Value",
 					Effect:   "Effect",
 				}},
-				SecondsOfInactivityBeforeIdling:     pointer.Int32Ptr(1800),
-				SecondsOfRunBeforeIdling:            pointer.Int32Ptr(-1),
+				SecondsOfInactivityBeforeIdling: pointer.Int32Ptr(1800),
+				SecondsOfRunBeforeIdling:        pointer.Int32Ptr(-1),
+				EditorDownloadUrls: []chev2.EditorDownloadUrl{
+					{
+						Editor: "JetBrains",
```

Comment:
what are the other valid options / use-cases for configuring download URLs e.g. semantically, what are the other possible values for the "Editor" property, even theoretically for the future? my understanding is that you are trying to make it more generic, but this structure is very confusing for me in the current state